sections/main-password-header.liquid

```diff
@@ -1,15 +1,21 @@
+<style type="text/css">
+  .password-logo {
+    max-width: {{ section.settings.logo_max_width }}px
+  }
+</style>
+
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="password-header">
     {%- if section.settings.logo -%}
-      <img
-        src="{{ section.settings.logo | image_url: width: 500, height: 500 }}"
-        class="password-logo"
-        alt="{{ section.settings.logo.alt | default: shop.name | escape }}"
-        style="max-width: {{ section.settings.logo_max_width }}px"
-        width="{{ section.settings.logo_max_width }}"
-        height="{{ section.settings.logo_max_width | divided_by: section.settings.logo.aspect_ratio }}"
-        loading="lazy"
-      >
+      {%- assign logo_alt = section.settings.logo.alt | default: shop.name | escape -%}
+      {% assign logo_height =  section.settings.logo_max_width | divided_by: section.settings.logo.aspect_ratio %}
+      {{ section.settings.logo | image_url: width: section.settings.logo.width | image_tag: 
+        loading: 'lazy', 
```

Comment:
Same here wondering if we need lazy or not 🤔

Reply:
Same as here: https://github.com/Shopify/dawn/pull/1701#discussion_r875185778